src/App.js

```diff
@@ -6,14 +6,12 @@ import users from './api/users';
 
 import { Todo } from './components/Todo/Todo';
 
-const preparedTodos = todos.map(todo => ({
-  name: users[todo.userId - 1].name,
-  username: users[todo.userId - 1].username,
-  email: users[todo.userId - 1].email,
-  title: todo.title,
-  completed: todo.completed,
-  id: todo.id,
-}));
+const preparedTodos = todos.map(todo => (
+  {
+    ...users.find(current => current.id === todo.userId),
```

Comment:
```suggestion
    user: users.find(current => current.id === todo.userId),
```
user and todo both have an `id` field, in this case resulting object does have only one id, from todo. This doesn't crash anything, but i don't sure you want that behaviour